### ISSUE TITLE
GraphQL Playground HTML does not contain a License

### ISSUE THREAD

**Issue by johannes-scharlach** [prismagraphql/graphql-playground#746]
#### This issue pertains to the following package(s):

- [ ] GraphQL Playground - Electron App
- [x] GraphQL Playground HTML
- [ ] GraphQL Playground
- [ ] GraphQL Playground Express Middleware
- [ ] GraphQL Playground Hapi Middleware
- [ ] GraphQL Playground Koa Middleware
- [ ] GraphQL Playground Lambda Middleware

#### What OS and OS version are you experiencing the issue(s) on?
n/a

#### What version of graphql-playground(-electron/-middleware) are you experiencing the issue(s) on?
latest

#### What is the expected behavior?
There should be a LICENSE file in the package (or at least a license mentioned in the package.json). Judging by the other packages this should be MIT.

#### What is the actual behavior?
Neither a LICENSE file is present, nor a license specified in package.json

#### What steps may we take to reproduce the behavior?
n/a

**Comment by jpeddicord** [comment]
Just noticed this as well; the child packages in this repository all state "SEE LICENSE IN LICENSE" where no license file is present, relative to their package.jsons.

From the content of the LICENSE file at the repo root, these could all probably just say `"license": "MIT"`.

**Comment by codepunkt** [comment]
The "license" field in `package.json` is supposed to contain a [spdx](https://spdx.org/) license identifier or license expression - which in this case, would be "MIT". License automation tools like [webpack-license-plugin](https://github.com/codepunkt/webpack-license-plugin) might warn or bail when encountering something else.

Aside from that, every OSS license not only comes with permissions and conditions, but also with _obligations_. One obligation of the "MIT" license is that you have to provide a copy of the license and copyright notice when using MIT licensed software.
Everyone using graphql-playground packages and distributing the software built on top of it to customers __has__ to provide both of them - the license text and the copyright notice.

This requires a `LICENSE` file with the MIT license text and copyright notice to be included in every npm package release. Because there is no direct link between git commits on a multi-package repository and releases on npm, it is not enough to have this LICENSE file somehwere in the repo (such as the repo root).

[lerna](https://lernajs.io/) automates this process in newer versions and takes the `LICENSE` file from the monorepo root and includes it on publishing packages.

---
TLDR

- [ ] Change "licene" field of every package to "MIT"
- [ ] Include LICENSE file from root of repository that contains the MIT license text and a copyright notice on every npm publish of every package
